corehq/apps/data_interfaces/tasks.py

```diff
@@ -1,13 +1,30 @@
 import uuid
+
 from celery.task import task
+from celery.utils.log import get_task_logger
 from django.core.cache import cache
-from corehq.apps.data_interfaces.utils import add_cases_to_case_group
+from django.template.loader import render_to_string
 from soil import CachedDownload
 
+from corehq.apps.data_interfaces.utils import add_cases_to_case_group, archive_forms
+from dimagi.utils.django.email import send_HTML_email
+
+logger = get_task_logger('data_interfaces')
+ONE_HOUR = 60 * 60
+
 
 @task
 def bulk_upload_cases_to_group(download_id, domain, case_group_id, cases):
     results = add_cases_to_case_group(domain, case_group_id, cases)
     temp_id = uuid.uuid4().hex
-    expiry = 60 * 60
-    cache.set(download_id, results, expiry)
+    cache.set(download_id, results, ONE_HOUR)
+
+
+@task
+def bulk_archive_forms(user, uploaded_data):
+    response = archive_forms(user, uploaded_data)
+    [logger.info("[Data interfaces] {}".format(msg)) for msg in response['errors']]
+    [logger.info("[Data interfaces] {}".format(msg)) for msg in response['success']]
```

Comment:
Got it. Yea I thought it was excessive to spend the two lines on something so trivial. Actually I was surprised Python didn't let me leave off the brackets on the list comprehension. Will make the change. And the delayed string interpolation is a nice trick.